spec/pandoc_parser_spec.rb

```diff
@@ -1,63 +1,35 @@
 require 'spec_helper'
+require 'pandoc-ruby'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.